documentation/jetty/modules/operations-guide/pages/deploy/index.adoc

```diff
@@ -372,6 +379,10 @@ $ java -jar $JETTY_HOME/start.jar -Dwebapps.dir=/opt/myapps
 
 Properties specified in environment properties file (described in <<environment-configuration,this section>>) and in the web application properties file (described in <<web-application-files,this section>>) are available when processing the Jetty context XML file.
 
+Remember that Jetty Context XML files in the `$JETTY_BASE/webapps` directory can only refer to classes present in the environment class-path or module-path, and in the server class-path or module-path.
```

Comment:
```suggestion
Remember that Jetty Context XML files in the `$JETTY_BASE/webapps` directory can only refer to classes present in the environment class-path (or module-path), and in the server class-path (or module-path).
```

Reply:
This section goes into far more detail than the similar (new) section below talking about com.acme.MyListenerFeature, perhaps we need a link (in both of these places) to new section on how classes are found (the environment classloader, the classpath, the modulepath, etc)

Reply:
I added a link pointing to the class loader hierarchy section.